### PR TITLE
updating styles for list view

### PR DESCRIPTION
**Pull request by Jmumenthaler**

### CODE REVIEW

File: src/components/Songs.jsx [hunk 1]
```diff
@@ -41,61 +43,93 @@ export default class Songs extends Component {
...
-                })}
+                <div className="filter-navigation">
+                    <div className="filter-navigation-inner">
+                        <ul className="filter-size">
+                            <li>Size</li>
+                            {this.renderSizes()}
+                        </ul>
+                    </div>
+                </div>
+                <div className={"song-list-container " + sizeClass}>
```
Comment (FotoVerite) [inline src/components/Songs.jsx]: Interpolation might be a better choice here and a few other places but I also see how that gets confusing in jsx based classes. 
{"song-list-container " + sizeClass} vs {`song-list-container ${sizeClass}`} is kinda a tossup on readability.

**Comment by FotoVerite** [comment]
Please remove the build/2008 from the repo.

File: src/components/Songs.jsx (hunk 1 above)
Comment (Jmumenthaler) [reply src/components/Songs.jsx]: yeah, prob.

Comment (Jmumenthaler) [reply src/components/Songs.jsx]: Ah shit, maybe this needs lots more work actually.

Comment (FotoVerite) [reply src/components/Songs.jsx]: Nah, it's not that big of a deal. As I said it's a tossup but we should figure out a style and code guide at some point to decide these issues.

**Pull request merged**